src/components/User/UserBalanceStats.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+
+import { Size } from 'decentraland-gatsby/dist/components/Props/types'
+import useAsyncMemo from 'decentraland-gatsby/dist/hooks/useAsyncMemo'
+import { Link } from 'decentraland-gatsby/dist/plugins/intl'
+import TokenList from 'decentraland-gatsby/dist/utils/dom/TokenList'
+import profiles, { Profile } from 'decentraland-gatsby/dist/utils/loader/profile'
+import { Header, HeaderProps } from 'decentraland-ui/dist/components/Header/Header'
+import { Loader } from 'decentraland-ui/dist/components/Loader/Loader'
+
+import './UserBalanceStats.css'
+import Username from './Username'
+
+export type UserBalanceStatsProps = {
+  size?: HeaderProps['size']
+  to?: string
+  className?: string
+  address?: string
+  disabled?: boolean
+  sub?: boolean
```

Comment:
What is sub?

Reply:
This is an old component I renamed to give way to the new UserStats. We can delete it, it's only used in Balance.